formulae/nemo.py

```diff
@@ -539,25 +541,42 @@ def r_multipassage(self, objectIds, subreferences, lang=None):
         #ids contient tous les differents objectId
         ids = objectIds.split('+')
         translations = {}
+        view=1
         for i in ids:
+            if "manifest" in i:
+                i = re.sub(r'^manifest:', '', i)
             p = self.resolver.getMetadata(self.resolver.getMetadata(i).parent.id)
             translations[i] = [m for m in p.readableDescendants if m.id not in ids]
         passage_data = {'template': 'main::multipassage.html', 'objects': [], "translation": translations}
         subrefers = subreferences.split('+')
         for i, id in enumerate(ids):
             if self.check_project_team() is True or id in self.open_texts:
+                v = False
+                if "manifest:" in id:
+                    id = re.sub(r'^manifest:', '', id)
+                    v = True
                 if subrefers[i] in ["all", 'first']:
                     subref = self.get_reffs(id)[0][0]
                 else:
                     subref = subrefers[i]
                 d = self.r_passage(id, subref, lang=lang)
                 del d['template']
-                d["IIIFviewer"] = id in self.app.picture_file
-                if 'previous_search' in session:
-                    result_sents = [x['sents'] for x in session['previous_search'] if x['id'] == id]
-                    if result_sents:
-                        d['text_passage'] = self.highlight_found_sents(d['text_passage'],
-                                                                       self.convert_result_sents(result_sents))
+                if v == True:
+                    formulae = self.app.picture_file["manifest:"+id]
+                    d["objectId"] = "manifest:"+id
+                    d["div_v"] = "manifest"+str(view)
+                    view = view+1
```

Comment:
For lines 565-568, always put spaces around the `+`. This is the best practice in Python to always have spaces around mathematical operators (+, -, /, *, =, etc.).